# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: accessible/tests/crashtests/crashtests.list
# Commit: 61e813c94bc9
# Full Hash: 61e813c94bc98cd2ebb03113fb3c1c1bf02283c8
# Author: James Teh <jteh@mozilla.com>
# Date: 2019-10-30 15:06:46
# Description:
#   Bug 1585851: When processing a11y insertions, don't try to move an accessible if its new previous sibling hasn't been moved into its new container yet. r=MarcoZ
#   
#   Sometimes, depending on how children were changed, children might be in the insertion list out of order; e.g. [child2, child1].
#   It's also possible that an earlier child (child1 in the above example) is being moved out of another container.
#   When processing the earlier insertion (child2), we'll determine we need to move it within its parent and will fetch its new previous sibling so we can move it into the right place.
# ==============================================================================

diff -r 1ee997d2152a -r 61e813c94bc9 accessible/tests/crashtests/crashtests.list
--- a/accessible/tests/crashtests/crashtests.list	Tue Oct 29 18:15:28 2019 +0000
+++ b/accessible/tests/crashtests/crashtests.list	Wed Oct 30 06:32:50 2019 +0000
@@ -13,6 +13,7 @@
 load 1494707.html
 load 1503964.html
 load 1415667.html
+load 1585851.html
 
 # last_test_to_unload_testsuite.xhtml MUST be the last test in the list because it
 # is responsible for shutting down accessibility service affecting later tests.
